lib-multisrc/zeistmanga/build.gradle.kts

```diff
@@ -2,4 +2,4 @@ plugins {
     id("lib-multisrc")
 }
 
-baseVersionCode = 9
+baseVersionCode = 10
```

Comment:
Since there's no real change in the base class, it should be fine to leave this as-is

Reply:
agreed, no functionality changed in the base class so no need to bump it